code_samples/data/chunking_evaluation/ground_truth/qa_dataset.csv

```diff
@@ -0,0 +1,59 @@
+id,question,answer,source
```

Comment:
where is this dataset from?

Reply:
We created this compact Q&A dataset manually from the public [MSFT solutions playbook](https://microsoft.github.io/code-with-engineering-playbook/machine-learning/ml-data-exploration/) documents, extracting 4 documents as examples.